DirectProgramming/DPC++/CombinationalLogic/mandelbrot/License.txt

```diff
@@ -1,4 +1,4 @@
-Copyright Intel Corporation
+Copyright 2019 Intel Corporation
```

Comment:
The standard has been changed to not including the date. Please remove this from PR